src/Bicep.Core/Semantics/NameBindingVisitor.cs

```diff
@@ -398,6 +398,17 @@ private ErrorSymbol AmbiguousSymbol<S>(IdentifierSyntax signifier, IEnumerable<S
         private IEnumerable<Symbol> FindFunctionMatchesInAllNamespaces(IdentifierSyntax identifierSyntax)
             => namespaceResolver.ResolveUnqualifiedFunction(identifierSyntax, includeDecorators: allowedFlags.HasAnyDecoratorFlag());
 
+        public override void VisitAssertDeclarationSyntax(AssertDeclarationSyntax syntax)
```

Comment:
>VisitAssertDeclarationSyntax [](http://example.com/codeflow?start=29&length=28)

nit - This method should be above with the rest of the public methods. They're all very similar, so it makes it easy to compare and read.